src/main/java/ch/njol/skript/util/DelayedChangeBlock.java

```diff
@@ -262,6 +262,11 @@ public BlockFace getFace(final Block block) {
 	@Override
 	public BlockState getState() {
 		return b.getState();
+	}	
+
+	@Override
+	public BlockState getState(boolean bool) {

```

Comment:
how come you changed these methods? this won't work on normal spigot (the method you call does not exist there). Also, you should name the parameter `useSnapshot` instead of `bool` if this is kept

Reply:
Actually i were using Spigot 1.8.8 while testing this PR, i tried again and there is no any error in the console. hmm... Also i don't know how to prevent calling them if doesn't exist.

Reply:
maybe it was removed on newer versions? it isn't in the spigot javadocs, only papers

Reply:
but that doesn't explain why this change is in this pr anyways

Reply:
Well it is added in PaperSpigot 1.12.**2** but doesn't give error on Spigot 1.8.8 i don't know.

Oh yeah i forgot to explain that, because i used the new [hide/showPlayer(Plugin, Player)](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/Player.html#hidePlayer-org.bukkit.plugin.Plugin-org.bukkit.entity.Player-) method that comes with 1.12.**2** if it exists 🤷‍♀️

Reply:
Yeah they don't call a new method just creates.